### DISCUSSION TITLE
Country Availability

### DISCUSSION THREAD

**Discussion by mohammadshaheer4** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#60] [2.0 Getting your free Azure credits]
Hi!

I just wanted to ask that whether Azure free student subscription is available in the Qatar region or not? If not, what other alternatives I have?

**Comment by EY-glowglowglow** [comment]
Hi there,
Please refer to the FAQ here - https://azure.microsoft.com/en-us/free/free-account-students-faq/
